tests/build/chains.go

```diff
@@ -1,7 +1,6 @@
 package build
```

Comment:
seems the commenting out of functions in this file should be undone, no?

Reply:
it's required due to a changes been made in EnterpriseContractPolicy api , 
contract team can fix it, meanwhile I had to comment these lines to make it pass